sdks/java/io/snowflake/src/test/java/org/apache/beam/sdk/io/snowflake/test/unit/credentials/ExternalReadCredentialsTest.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I wonder if those tests are actually still needed

Reply:
Why do you thinks so ?

Reply:
Because they test SnowflakeCredentialsFactory.createCredentials(c) which is already tested elsewhere. But they also test creation of the transform, so maybe let's leave them

Reply:
I leave them .